### ISSUE TITLE
LoadLibrary failure:  %1 is not a valid Win32 application. (eseis.dll)

### ISSUE THREAD

**Issue by rozsasarpi** [coffeemuggler/eseis#2]
I am having a problem installing the package. 

After issuing this command:
`devtools::install_github(repo = "coffeemuggler/eseis", ref = "0.3.1")`

The following error is received:
```
Error in inDL(x, as.logical(local), as.logical(now), ...) : 
  unable to load shared object 'C:/Users/arpada/Documents/R/win-library/3.3/eseis/libs/x64/eseis.dll':
  LoadLibrary failure:  %1 is not a valid Win32 application.
```

Any suggestion how to resolve the problem? Let me know if you need additional information about the nature of the problem. Thank you in advance.

The full message after calling `devtools::install_github(repo = "coffeemuggler/eseis", ref = "0.3.1")`
```
* installing *source* package 'eseis' ...
** libs
c:/Rtools/mingw_64/bin/g++ -m64 -shared -s -static-libgcc -o eseis.dll tmp.def RcppExports.o stalta_event_freeze.o stalta_event_nofreeze.o -LC:/swarm/workspace/External-R-3.3.2/vendor/extsoft/lib/x64 -LC:/swarm/workspace/External-R-3.3.2/vendor/extsoft/lib -LC:/PROGRA~1/MIE74D~1/MRO-33~1.2/bin/x64 -lR
C:/Rtools/mingw_64/bin/../lib/gcc/x86_64-w64-mingw32/4.9.3/../../../../x86_64-w64-mingw32/bin/ld.exe: RcppExports.o: invalid relocation type 42
C:/Rtools/mingw_64/bin/../lib/gcc/x86_64-w64-mingw32/4.9.3/../../../../x86_64-w64-mingw32/bin/ld.exe: BFD (GNU Binutils) 2.25 assertion fail ../../../src/binutils-2.25/bfd/elf64-x86-64.c:341
C:/Rtools/mingw_64/bin/../lib/gcc/x86_64-w64-mingw32/4.9.3/../../../../x86_64-w64-mingw32/bin/ld.exe: RcppExports.o: invalid relocation type 42
C:/Rtools/mingw_64/bin/../lib/gcc/x86_64-w64-mingw32/4.9.3/../../../../x86_64-w64-mingw32/bin/ld.exe: BFD (GNU Binutils) 2.25 assertion fail ../../../src/binutils-2.25/bfd/elf64-x86-64.c:341
C:/Rtools/mingw_64/bin/../lib/gcc/x86_64-w64-mingw32/4.9.3/../../../../x86_64-w64-mingw32/bin/ld.exe: RcppExports.o: invalid relocation type 42
C:/Rtools/mingw_64/bin/../lib/gcc/x86_64-w64-mingw32/4.9.3/../../../../x86_64-w64-mingw32/bin/ld.exe: BFD (GNU Binutils) 2.25 assertion fail ../../../src/binutils-2.25/bfd/elf64-x86-64.c:341
C:/Rtools/mingw_64/bin/../lib/gcc/x86_64-w64-mingw32/4.9.3/../../../../x86_64-w64-mingw32/bin/ld.exe: RcppExports.o: invalid relocation type 42
C:/Rtools/mingw_64/bin/../lib/gcc/x86_64-w64-mingw32/4.9.3/../../../../x86_64-w64-mingw32/bin/ld.exe: BFD (GNU Binutils) 2.25 assertion fail ../../../src/binutils-2.25/bfd/elf64-x86-64.c:341
C:/Rtools/mingw_64/bin/../lib/gcc/x86_64-w64-mingw32/4.9.3/../../../../x86_64-w64-mingw32/bin/ld.exe: RcppExports.o: invalid relocation type 42
C:/Rtools/mingw_64/bin/../lib/gcc/x86_64-w64-mingw32/4.9.3/../../../../x86_64-w64-mingw32/bin/ld.exe: BFD (GNU Binutils) 2.25 assertion fail ../../../src/binutils-2.25/bfd/elf64-x86-64.c:341
RcppExports.o: In function `Rcpp::PreserveStorage<Rcpp::Vector<14, Rcpp::PreserveStorage> >::PreserveStorage()':
/home/mdietze/R/x86_64-pc-linux-gnu-library/3.3/Rcpp/include/Rcpp/storage/PreserveStorage.h:10: Cannot get section contents - auto-import exception
collect2.exe: error: ld returned 5 exit status
installing to C:/Users/arpada/Documents/R/win-library/3.3/eseis/libs/x64
** R
** data
** preparing package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded
Error in inDL(x, as.logical(local), as.logical(now), ...) : 
  unable to load shared object 'C:/Users/arpada/Documents/R/win-library/3.3/eseis/libs/x64/eseis.dll':
  LoadLibrary failure:  %1 is not a valid Win32 application.

Error: loading failed
Execution halted
ERROR: loading failed
* removing 'C:/Users/arpada/Documents/R/win-library/3.3/eseis'
Error: Command failed (1)
```

version information:

```
platform       x86_64-w64-mingw32          
arch           x86_64                      
os             mingw32                     
system         x86_64, mingw32             
status                                     
major          3                           
minor          3.2                         
year           2016                        
month          10                          
day            31                          
svn rev        71607                       
language       R                           
version.string R version 3.3.2 (2016-10-31)
nickname       Sincere Pumpkin Patch
```

**Comment by coffeemuggler** [comment]
Dear Árpád,

Thanks for the note. I tried yesterday to install the latest developer 
version of eseis to a Mac OSX system, also via devtools. There, I found 
similar problems. Seems to be either related to devtools or Rcpp (see 
https://github.com/ropensci/git2r/issues/246). Though, I think you are 
working on a Windows machine, spotting Rtools.

Did you try to install from local? If you go to my website 
(http://micha-dietze.de/pages/eseis.html) there is a Windows version of 
eseis (0.3.1). Try downloading and installing it.

Let me know if this worked, or not.

Micha
